pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -222,7 +223,7 @@ class RichProgressBar(ProgressBarBase):
 
     def __init__(
         self,
-        refresh_rate_per_second: int = 10,
+        refresh_rate: int = 1,
```

Comment:
This is a breaking change. Removing `refresh_rate_per_second` should follow the deprecation process before its removal

Reply:
I and Sean discussed about it, it would be really messy and unintuitive to support both the parameters for RichProgressBar. Also, as we do manual refresh now in this PR

Reply:
Also, 

I would recommend the following for the default value:

```py
import os
from typing import Optional

class TQDMProgressBar(ProgressBarBase):

    def __init__(self, refresh_rate: Optional[int] = None, process_position: int = 0):
        super().__init__()

        self._refresh_rate = self._resolve_refresh_rate(refresh_rate)

    # moved in the `ProgressBarBase`
    @staticmethod
    def _resolve_refresh_rate(refresh_rate: Optional[int]) -> int:
        if refresh_rate is None:
            refresh_rate = 20 if os.getenv("COLAB_GPU", "1") == "1" else 1
        return refresh_rate
```

IMO, the progress bar should be responsible to set up properly for colab.